.php_cs.dist

```diff
@@ -4,7 +4,7 @@ return PhpCsFixer\Config::create()
     ->setRules(array(
```

Comment:
Symfony uses https://github.com/FriendsOfPHP/PHP-CS-Fixer instead of phpcs. To be honest, I'm not entirely sure of the benefits and differences between them. PHPStorm doesn't have inline support for php-cs-fixer, but Symfony's current rules are for php-cs-fixer.

If you look at .circleci/config.yml, you'll see the command we use for this.